Projects/NET/NET04/NET04-AWS100.md

```diff
@@ -1,28 +1,31 @@
-# NET04-AWS100 — Create three Billing Alarms
+# NET04-AWS100 — Host a simple static webpage with S3 and CloudFront
 
 ## Cloud Service Provider
 * Amazon Web Services (AWS)
 
 ## Difficulty
 Level 100 (Introductory)
 
+Estimated time: 1 hour
```

Comment:
Format:
### Estimated Time
1 hour

-----
Great idea. Should be added to the project template as well #6

Reply:
Definitely a better format.